app/index.html

```diff
@@ -21,7 +21,12 @@
             skipStartupTypeset: true
         });
     </script>
+    <link href="jquery.tagsinput.css" rel="stylesheet" type="text/css" />
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/dropzone/5.1.1/min/basic.min.css" rel="stylesheet" type="text/css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/dropzone/5.1.1/min/dropzone.min.css" rel="stylesheet" type="text/css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/dropzone/5.1.1/min/dropzone.min.js"></script>
```

Comment:
These script-tag style import of dependencies doesn't look right. Talked to @baylee-d and it seems that Dropzone.js does not work without these imports. Same thing with tagsinput. Somehow `ember-osf-web` is also importing Dropzone.js this way, in addition to including them in package.json.

Reply:
Ember-cli-dropzonejs could be a future solution.